### ISSUE TITLE
`input` must not start with a slash when using `prefixUrl`

### ISSUE THREAD

**Issue by JounQin** [sindresorhus/got#1869]
#### What would you like to discuss?

```ts
got('//at.alicdn.com/t/font_1626314_1ym8doilp1d.js')
```

#### Checklist

- [x] I have read the documentation.

---

The error message seems not clear.

**Comment by szmarczak** [comment]
This is fixed in beta. Please fill in the issue template properly next time.

**Comment by JounQin** [comment]
@szmarczak 

> This is fixed in beta. Please fill in the issue template properly next time.

![image](https://user-images.githubusercontent.com/8336744/133109283-50395b35-cefc-4ce6-bf21-21ceb2a18a59.png)

**Comment by szmarczak** [comment]
Ah, this should be a bug report since you didn't provide a `prefixUrl`.

The message in Got 12 says that you mustn't provide a leading slash at all.

**Comment by JounQin** [comment]
I think error is acceptable but the error message is not clear, so I raised this issue as a question instead of a bug.

**Comment by szmarczak** [comment]
```
`input` must not start with a slash
```

this is very clear. Your input is `//at.alicdn.com/t/font_1626314_1ym8doilp1d.js` and it starts with a slash, hence the error.

**Comment by JounQin** [comment]
The confusing part is `prefixUrl`...

Let's don't waste time on this...😂

**Comment by szmarczak** [comment]
> The confusing part is prefixUrl...

It's a Got option.

> Ah, this should be a bug report since you didn't provide a prefixUrl.

Let me rephrase: you didn't provide a `prefixUrl`, yet still got an error saying you did. The `prefixUrl` part is removed from the message in Got 12.
